checkov/secrets/git_history_store.py

```diff
@@ -122,7 +122,15 @@ def get_added_and_removed_commit_hash(self, key: str, secret: PotentialSecret) -
         """
         try:
             secret_key = get_secret_key(secret.filename, secret.secret_hash, secret.type)  # by value type
-            enriched_secrets = self.secrets_by_file_value_type[secret_key]
+            enriched_secrets = self.secrets_by_file_value_type.get(secret_key, None)
```

Comment:
The default behavior is assiging `None`
```suggestion
            enriched_secrets = self.secrets_by_file_value_type.get(secret_key)
```